locales.software/1038-hu.cfos.txt

```diff
@@ -91,14 +91,14 @@ pnet_conn_local_ip = "Helyi IP cím %?";
 pnet_conn_public_ip = "Nyílvános IP cím %?";
 pnet_pubdir_header = "Nyílvános könyvtár";
 pnet_pubdir_subheader = "Az a könyvtár, amelyik tartalmazza a letölthető fájlokat.";
-pnet_pubdir_info = "Kérlek, add meg azt a mappa helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
+pnet_pubdir_info = "Kérlek, add meg annak a mappának a helyét, ahol azok a fájlok találhatók, amelyeket elérhetővé szeretnél tenni a letöltéshez.";
 smtp_header = "SMTP-szerver";
 smtp_subheader = "SMTP-szerver beállítása.";
 dynip_header = "Dinamikus IP-szerver";
 dynip_subheader = "Dinamikus IP-szerver beállítása";
 dlg_start_now_as_current_user = "Futtasd a %p-t, a jelenlegi felhasználóval";
 dlg_autostart = "%p indítása a Windows-zal együtt";
-dlg_autostart_service = "Telepítsd a %p, mint rendszer folyamat és bekapcsolásnál induljon el a Windows-al együtt";
+dlg_autostart_service = "Telepítés (%p) mint rendszer folyamat és bekapcsolásnál induljon el a Windows-zal együtt";
```

Comment:
Well, yes, you should use an "a" before "%p" in Hungarian if this is the
case.
I did not mention the other reason which is the inflexion in Hungarian, so
you should use it like:

"Telepítsd a %p-t"

It seems I should check the whole translation once more, to make
sure it is not broken grammatically.





2015. december 6. 17:47 Serj Lavrin írta, <notifications@github.com>:

> In locales.software/1038-hu.cfos.txt
> <https://github.com/cFos/l10n/pull/15#discussion_r46772685>:
>
> >  smtp_header = "SMTP-szerver";
> >  smtp_subheader = "SMTP-szerver beállítása.";
> >  dynip_header = "Dinamikus IP-szerver";
> >  dynip_subheader = "Dinamikus IP-szerver beállítása";
> >  dlg_start_now_as_current_user = "Futtasd a %p-t, a jelenlegi felhasználóval";
> >  dlg_autostart = "%p indítása a Windows-zal együtt";
> > -dlg_autostart_service = "Telepítsd a %p, mint rendszer folyamat és bekapcsolásnál induljon el a Windows-al együtt";
> > +dlg_autostart_service = "Telepítés (%p) mint rendszer folyamat és bekapcsolásnál induljon el a Windows-zal együtt";
>
> Particularly p% placeholder always upholds English program name.
>
> Just look into section name to determinate which program name will be
> there.
>
> For example, here we have:
>
> / ----------------------------------------------------------------------
> /
> /  6link / broadband connect / pnet
> /
> / ----------------------------------------------------------------------
>
> So we can see that %p will be replaced with cFos IPv6 Link, cFos
> Broadband Connect or cFos Personal Net (those apps share those strings),
> as per http://www.cfos.de/en/index.htm products.
>
> To be more exact, in that string will be used cFos Personal Net.
>
> Will it help to write article before them properly?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cFos/l10n/pull/15/files#r46772685>.
>